### PR TITLE
Issue1004 init co2

### LINKED ISSUE #1004: initial state of CO2 concentration
The initial state of the CO2 concentration is currently 0, which should be a reasonable value instead.

### PR DESCRIPTION
**Pull request by Mathadon**
this closes #1004

### CODE REVIEW

File: IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo [hunk 1]
```diff
@@ -139,6 +148,11 @@ equation
 <ul>
 <li>
 February 14, 2019 by Filip Jorissen:<br/>
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo]:
```suggestion
March 29, 2019 by Filip Jorissen:<br/>
```

**Comment by Mathadon** [comment]
@damienpicard can you review and merge this?

**Comment by damienpicard** [comment]
it would be nice to add a comment to mention what the assumption is behind this start value.

File: IDEAS/Buildings/Examples/InternalGainExample.mo [hunk 2]
```diff
@@ -35,6 +34,11 @@ equation
 </html>", revisions="<html>
 <ul>
 <li>
+January 25, 2019 by Filip Jorissen:<br/>
```
Comment (Mathadon) [inline IDEAS/Buildings/Examples/InternalGainExample.mo]:
```suggestion
April 10, 2019 by Filip Jorissen:<br/>
```

File: IDEAS/BoundaryConditions/Interfaces/PartialSimInfoManager.mo [hunk 3]
```diff
@@ -468,6 +471,11 @@ equation
 <ul>
 <li>
 January 25, 2019 by Filip Jorissen:<br/>
+Corrected molar mass fraction for consistency.
+See <a href=https://github.com/open-ideas/IDEAS/issues/1004>#1004</a>.
+</li>
+<li>
+January 25, 2019 by Filip Jorissen:<br/>
```
Comment (Mathadon) [inline IDEAS/BoundaryConditions/Interfaces/PartialSimInfoManager.mo]:
```suggestion
April 10, 2019 by Filip Jorissen:<br/>
```

**Pull request merged**